package.json

```
@@ -13,12 +13,15 @@
     "cache:clear": "rm -rf .cache",
     "evals": "npm run build-dom-scripts && tsx evals/index.eval.ts",
     "build-dom-scripts": "tsx lib/dom/genDomScripts.ts",
-    "build": "npm run build-dom-scripts && tsup lib/index.ts --dts",
-    "release": "changeset publish"
+    "build-types": "tsc --emitDeclarationOnly --outDir dist",
+    "build-js": "esbuild lib/index.ts --bundle --platform=node --outdir=dist --format=esm --minify --tree-shaking=true --packages=external",
```

i'd recommend against minifying; i think it's important to keep the dist readable, debuggable. users can always minify the package if they're bundling before deploying to their production environment if size is a concern

```suggestion
    "build-js": "esbuild lib/index.ts --bundle --platform=node --outdir=dist --format=esm --tree-shaking=true --packages=external",
```